### PR TITLE
Improve `-printpriority` output to log the modified fee, conventional fee, size, logical actions, and unpaid actions

### PR DESCRIPTION
**Pull request by daira**
This reflects the changes to de-emphasise fee rate. Also improve the `show_help.py` RPC test to include testing of `--help -help-debug`.

### CODE REVIEW

**Comment by daira** [comment]
Moving to draft because I want to improve it further to add the number of logical actions (and the number unpaid if any).

File: doc/release-notes.md [hunk 1]
```diff
@@ -4,83 +4,4 @@ release-notes at release time)
 Notable changes
 ===============
 
-RPC Changes
```
Comment (nuttycom) [inline doc/release-notes.md:7]: These changes should not be included on the `version-5.5.0` branch. Please revert them.

**Review by nuttycom — CHANGES_REQUESTED** [review]

File: doc/release-notes.md (hunk 1 above)
Comment (nuttycom) [reply doc/release-notes.md:7]: Actually, this just needs to be rebased, I think.

File: qa/rpc-tests/show_help.py [hunk 2]
```diff
@@ -49,6 +49,13 @@
        Execute command when the best block changes (%s in cmd is replaced by
        block hash)
 
+|  -blocksonly
```
Comment (nuttycom) [inline qa/rpc-tests/show_help.py:52]: Why are all these added, and why do they have leading `|` symbols?

Comment (daira) [reply qa/rpc-tests/show_help.py:52]: This is testing the output of `zcashd --help -help-debug`; see the first commit.

**Review by nuttycom — APPROVED** [review]
utACK

**Review by str4d — APPROVED** [review]
utACK

**Pull request merged**